libr/core/cmd_anal.c

```diff
@@ -109,6 +109,7 @@ static const char *help_msg_ae[] = {
 	"aesue", " [esil]", "step until esil expression match",
 	"aetr", "[esil]", "Convert an ESIL Expression to REIL",
 	"aex", " [hex]", "evaluate opcode expression",
+	"aexn", "[X] [N]  ", "emulate N instr from offset X",
```

Comment:
aex is to emulate an esil expression, this is more related to stepping so put it under `aes`

Reply:
the issue was expecting to have this to handle the arguments passed to aes, instead of having to do 10aes, be able to do aes 10, but aes doesnt implies setting the program counter to any specific address, but we can handle this like you did as a 2nd arg. or maybe set a different subcommand of aes (aesp) to specify the program counter

Reply:
after thinking i think we should have `aes [n]` and `aesp [pc] [n]`. what do you think?

Reply:
Yeah , having two commands , sounds good to me !!!